gitspy/Repository.swift

```diff
@@ -17,4 +17,55 @@ struct Repository {
     let newStars: Int
     let totalStars: Int
     
+    // MARK: Network
+
+    static func repos(userId: String, completion: (parsedRepos: [Repository]?) -> Void) {
+        let userResource = Resource(pathComponent: "repos?user_id=" + userId) { (object) -> [Repository]? in
+            guard let dictionary = object as? [String: AnyObject],
+                let repos = dictionary["repos"] as? [[String: AnyObject]] else {
+                return nil
+            }
+            
+            return Repository.array(repos)
+        }
+        
+        Networker.sharedInstance.loadResource(userResource, callback: completion)
+    }
+    
+    static func watch(repoId: Int, completion: (status: Bool?) -> Void) {
+        let boolResource = Resource(pathComponent: "repos/" + String(repoId), method: "POST") { (object) -> Bool? in
+            guard let status = object["status"] as? String else {
+                return nil
+            }
+
+            return status == "ok"
+        }
+        
+        Networker.sharedInstance.loadResource(boolResource, callback: completion)
+    }
```

Comment:
Actually now Models have: 
1. the model itself
2. Parsing (that is actually an initializer)
3. Networking

While I agree that Networking could be too much..
1) it is good looking:
`Repository.watch(repoID)` > `SomeClass.instance.watch(repoID)`

2) Creating a class or whatever to handle networking of a specific model is really better than handle it in the model itself? Then I would rather have a unique class for all the endpoints (`XXXService` instead of : `MeService`, `RepositoryService`, ...)